index.js

```diff
@@ -127,8 +127,8 @@ module.exports = function ExpressOpenApi (_routePrefix, _doc, _opts) {
   middleware.callbacks = middleware.component.bind(null, 'callbacks')
 
   // Expose ui middleware
-  middleware.redoc = ui.serveRedoc(`${routePrefix}.json`, opts)
-  middleware.swaggerui = ui.serveSwaggerUI(`${routePrefix}.json`, opts)
+  middleware.redoc = (options) => ui.serveRedoc(`${routePrefix}.json`, options)
+  middleware.swaggerui = (options) => ui.serveSwaggerUI(`${routePrefix}.json`, options)
```

Comment:
Since this is a breaking change we might want to bump to 1.0.0 to release it (despite that this is pre 1.x and technically any feature can be breaking). I guess I am fine doing that, but I would like to just remove redoc before we do so that we can make both breaking changes together. Do you want to just PR that separately and we can land that one first and this one (updated to remove the redoc stuff) after?

Reply:
Oh, and one other thing. When we add Release Please to the mix it will use Conventional Commits to calculate the semver bump. Might want to get used to it and start using it in your commit messages. In this case it would be like `feat!: pass options to ui middleware`.

Reply:
I will make a PR to remove redoc in a few minutes.

Reply:
> I assume you are referring to this

Yep, and their GH action: https://github.com/google-github-actions/release-please-action